src/quote/wrapped/index.test.ts

```diff
@@ -30,6 +32,28 @@ describe('WrappedQuoteProvider()', () => {
     }
     const quoteProvider = new WrappedQuoteProvider(provider)
     const quote = await quoteProvider.getQuote(request)
-    expect(quote).toBeNull()
+    if (!quote) fail()
+    expect(quote.indexTokenAmount).toEqual(request.indexTokenAmount)
+    expect(quote.inputOutputTokenAmount.gt(0)).toEqual(true)
+  })
+
+  test('returns a quote for redeeming MMI', async () => {
+    const outputToken: QuoteToken = {
+      address: USDC.address!,
+      decimals: 6,
+      symbol: USDC.symbol,
+    }
+    const request: FlashMintWrappedQuoteRequest = {
+      isMinting: false,
+      inputToken: indexToken,
+      outputToken,
+      indexTokenAmount: wei(1),
+      slippage: 0.5,
+    }
+    const quoteProvider = new WrappedQuoteProvider(provider)
+    const quote = await quoteProvider.getQuote(request)
+    if (!quote) fail()
+    expect(quote.indexTokenAmount).toEqual(request.indexTokenAmount)
+    expect(quote.inputOutputTokenAmount.gt(0)).toEqual(true)
```

Comment:
Seems to work. Please check if the actual returned value of `inputOutputTokenAmount` makes sense.